py3status/modules/bluetooth.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Module for displaying bluetooth status
+
+Requires:
+    - hcitool
+    - sed
+
+@author jmdana <https://github.com/jmdana>
+@license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
+"""
+
+from time import time
+import subprocess
+import shlex
+
+class Py3status:
+    # configuration parameters
+    cache_timeout = 10
+    color_good = None
+    color_bad = None
+
+    def __init__(self):
+        pass
+
+    def kill(self, i3s_output_list, i3s_config):
+        pass
+
+    def on_click(self, i3s_output_list, i3s_config, event):
```

Comment:
__init__, kill and on_click should be removed if they don't actually do anything please